src/internal/operators/timeout.ts

```diff
@@ -330,7 +330,7 @@ export function timeout<T, O extends ObservableInput<any>, M>(
     // hand of the subscription to that one.
     let originalSourceSubscription: Subscription;
     // The subscription for our timeout timer. This changes
-    // every time get get a new value.
+    // every time get a new value.
```

Comment:
```suggestion
    // every time get gets a new value.
```
?

Reply:
It's quite common to use `we`, thus this could be: `every time we get a new value.`